modelarmor/api/ModelArmor.Samples.Tests/ModelArmorFixture.cs

```diff
@@ -15,53 +15,180 @@
+    }
 
-        // Create client.
-        Client = ModelArmorClient.Create();
+    // Creates a DLP Inspect Template and returns its name.
+    public string CreateInspectTemplate(string displayName = "Test Inspect Template")
+    {
+        var parent = new LocationName(ProjectId, LocationId).ToString();
+        var templateId = $"inspect-{GenerateUniqueId()}";
+        var request = new CreateInspectTemplateRequest
+        {
+            Parent = parent,
+            InspectTemplate = new InspectTemplate
+            {
+                DisplayName = displayName,
+                InspectConfig = new InspectConfig
+                {
+                    InfoTypes =
+                    {
+                        new InfoType { Name = "PERSON_NAME" },
+                        new InfoType { Name = "EMAIL_ADDRESS" },
+                        new InfoType { Name = "US_INDIVIDUAL_TAXPAYER_IDENTIFICATION_NUMBER" },
+                    },
+                },
+            },
+            TemplateId = templateId,
+        };
+        var response = DlpClient.CreateInspectTemplate(request);
+        return response.Name;
+    }
 
-        // Create a template name for quickstart test.
-        string templateId = $"test-csharp-{Guid.NewGuid().ToString("N").Substring(0, 8)}";
-        TemplateForQuickstartName = TemplateName.FromProjectLocationTemplate(
-            ProjectId,
-            LocationId,
-            templateId
-        );
+    // Creates a DLP Deidentify Template and returns its name.
+    public string CreateDeidentifyTemplate(string displayName = "Test Deidentify Template")
+    {
+        var parent = new LocationName(ProjectId, LocationId).ToString();
+        var templateId = $"deidentify-{GenerateUniqueId()}";
+        var request = new CreateDeidentifyTemplateRequest
+        {
+            Parent = parent,
+            DeidentifyTemplate = new DeidentifyTemplate
+            {
+                DisplayName = displayName,
+                DeidentifyConfig = new DeidentifyConfig
+                {
+                    InfoTypeTransformations = new InfoTypeTransformations
+                    {
+                        Transformations =
+                        {
+                            new InfoTypeTransformations.Types.InfoTypeTransformation
+                            {
+                                PrimitiveTransformation = new PrimitiveTransformation
+                                {
+                                    ReplaceConfig = new ReplaceValueConfig
+                                    {
+                                        NewValue = new Value { StringValue = "[REDACTED]" },
+                                    },
+                                },
+                            },
+                        },
+                    },
+                },
+            },
+            TemplateId = templateId,
+        };
+        var response = DlpClient.CreateDeidentifyTemplate(request);
```

Comment:
Similar to my previous comment, consider adding this to the clean up collection right here.